Adds support for custom pysmurf-controller docker service names

jlashner opened the pull request:
This PR adds support for setting docker service names for pysmurf-controller dockers. This will enable us to change docker services from `ocs-pysmurf-s<slot>` to `ocs-<instance-id>` by specifying manually the service names to use in the sys-config.

Related: https://github.com/simonsobs/ocs-deployment-configs/issues/126#issuecomment-2418366911

msilvafe commented:
I definitely see @BrianJKoopman's frustration with the current setup so thanks for addressing this but can you describe what testing you've done before I approve?

jlashner commented:
Ok, I moved the operations for starting / stopping the pysmurf-controllers to a separate function so that they could be called and tested outside of a full hammer.

I also added the subcommand `jackhammer controller <up/down> <slots>`  to make it easier to manage pysmurf-controllers without looking up their instance-ids.

This has been tested on satp1.

msilvafe reviewed (approved):
Great, thanks for testing!
